COLLABORATOR_GUIDE.md

```diff
@@ -73,6 +73,77 @@ All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
 
+## Breaking Changes
+
+Backwards-incompatible changes may land on the master branch at any time after
+sufficient review by collaborators and approval of at least two CTC members.
+
+Examples of breaking changes include, but are not necessarily limited to,
+removal or redefinition of existing API arguments, changing return values,
+adding or removing errors, altering the timing, and changing the non-internal
+side effects of using a particular API.
+
+With a few notable exceptions outlined below, when backwards incompatible
+changes to an API are necessary, the existing API *must* be Deprecated *first*
+and the new API either introduced in parallel or added after the next major
+Node.js version following the Deprecation as a replacement for the Deprecated
+API. In other words, existing API's *must not* change without a Deprecation.
```

Comment:
I'm getting flashbacks of my fourth grade teacher ;-) ... that dang `'` is my nemesis I swear